contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -143,24 +143,28 @@ <h2 id="contribute">Contribution Details</h2>
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
                                    placeholder="Give a list of admin names, separated by comma"
-                                   type="text" value="{{ post.contributionAdmins|join(', ') }}"></div>
+                                   type="text" value="{{ post.contributionAdmins|join(', ') }}">
                         {% else %}
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
                                    placeholder="Give a list of admin names, separated by comma"
-                                   type="text"></div>
+                                   type="text">
                         {% endif %}
-            </div>
-            <div class="form-group">
-                <label class="control-label col-sm-2">Contributor Details</label>
-                <div>
-                    <button class="button button2" id="add-contributor-person">Add Person</button>
-                    <button class="button button2" id="add-contributor-org">Add Organization</button>
+                    </div>
                 </div>
-            </div>
+
+                <div class="form-group">
+                    <label class="control-label col-sm-2">Contributor Details</label>
+                    <div class="col-sm-10">
+                        <button class="button button2" id="add-contributor-person" style="margin-left:0px">Add Person</button>
```

Comment:
The margin-left style could be removed as without it, it's getting aligned well within the right side column. I'm unsure if everything needs to start on the same vertical line. Of course, we don't need elements to look moving on to the left side of the line.

Reply:
updated. removed inline style.

Reply:
Thanks.